modules/jt-migrating-a-sample-pipeline-from-jenkins-to-openshift-pipelines.adoc

```diff
@@ -34,10 +35,11 @@ pipeline {
 }
 ----
 
-== Tekton pipeline
-In Tekton, the equivalent example of the Jenkins pipeline comprises of three tasks, each of which can be written declaratively using the YAML syntax:
+== OpenShift Pipelines pipeline
 
-.Example `build` task
+To create a pipeline in OpenShift Pipelines that is equivalent to the preceding Jenkins pipeline, you create the following three tasks:
```

Comment:
`you create the following three tasks ` replace by `the following three tasks can be created`

Reply:
Red Hat documentation prefers using the second-person pronoun, "you."